core/src/test/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookupJTSTest.java

```diff
@@ -60,11 +72,14 @@ public void testSmallScenario() {
     @Test
     public void testExact() {
```

Comment:
With the SpatialRuleLookupArray there was a parameter to switch to the exact lookup. The SpatialRuleLookupJTS is exact by default. Maybe it would make sense to rename the tests?

Reply:
Maybe something like `testPrecision` ?

Reply:
Yes sure, or maybe something like testCloseToBorder etc.

I am not sure we need to change it at all, but just thought it might be worth to consider.

Also the test was designed to catch some edge cases of the exact parameter, I am not sure we need it at all for the JTS implementation? Also there is [a comment](https://github.com/graphhopper/graphhopper/pull/1829/files/ebbb38bfe281a13e2a961eddb0ea6fb1438afb38#diff-f751d631a68af77814ca82af419acd9bL74) about the border tile that existed in the array lookup, but probably not with JTS.

Reply:
I think one test should be enough to enforce precision.

Reply:
I don't mean to add more tests. I was thinking of either a) removing this test, as it was designed to catch the edge cases involved with a tile based lookup with a precision of 1 or b) rewriting it a bit for example to test coordinates that are closer inside/outside of the polygon, so it's a precision test.